examples/twitter-monitor.yml

```diff
@@ -0,0 +1,47 @@
+# This integration does a post request to the twitter auth api, uses a token from the auth api to query the tweet api.
+# As an added benefit it then does a third query to analyses each tweets sentiment.
+# Check out https://newrelic.com/resources/webinars/your-observability-data-in-one-place1 for live demo.
+integrations:
+  - name: nri-flex
+    interval: 60m
+    config:
+      name: twitterInfo
+      #point lookup file at a json file that contains api key and api secret key
+      lookup_file: /home/ubuntu/apikeys.json
```

Comment:
please add a sample lookup JSON file to this PR to be submitted with this config.
ref: [example](https://github.com/newrelic/nri-flex/blob/master/examples/openssl-expiration-lookup.json)